metrics-jmx/src/main/java/com/codahale/metrics/jmx/JmxReporter.java

```diff
@@ -570,6 +571,7 @@ public void onCounterAdded(String name, Counter counter) {
             } catch (JMException e) {
                 LOGGER.warn("Unable to register counter", e);
             }
+            return null;
```

Comment:
Why don't we return the `Counter` which has been passed into the method when it was added?